src/constants.js

```diff
@@ -17,6 +17,8 @@ const IGNORE_PATHS = process.env["INPUT_IGNOREPATHS"]
   ? process.env["INPUT_IGNOREPATHS"].split(",")
   : ["node_modules/**"];
 
+const JPEG_PROGRESSIVE = process.env["INPUT_JPEGPROGRESSIVE"] === "true";
```

Comment:
My guess is you've done it this way because the env var will be a string, rather than boolean?

Reply:
@benschwarz yes, so it'll default to false, regardless of the string, as long as it's not equal to "true". 

I hope this is fine, since you wanted `false` as the default option?

Reply:
Yes, this is fine. Just wanted to make sure I understood it properly. 👍